### ISSUE TITLE
ETW Input Should Support .NET Standard

### ISSUE THREAD

**Issue by commonsensesoftware** [Azure/diagnostics-eventflow#242]
## Overview
The ETW input provider (Microsoft.Diagnostics.EventFlow.Inputs.Etw) should support .NET Standard as all of its dependencies also support .NET standard.

## Proposed Solution
Change [line 7](../blob/3f045399b0cffc63797a9bbbd9fa0c8e7d15637a/src/Microsoft.Diagnostics.EventFlow.Inputs.Etw/Microsoft.Diagnostics.EventFlow.Inputs.Etw.csproj#L7) to:

```xml
<TargetFrameworks>net451;netstandard1.6;netstandard2.0</TargetFrameworks>
```

**Comment by karolz-ms** [comment]
Hi @commonsensesoftware, thanks for good suggestion, I did not realize that since version 2.0.x TraceEvent supports .NET Core. Thanks for bringing this to my attention!

I would like to do a bit of testing before changing the target, but we should be able to make this change in the next week or two.

**Comment by karolz-ms** [comment]
Oops, just noticed the PR. Let's continue there

**Comment by commonsensesoftware** [comment]
I _thought_ they had, but I had to manually check too. Turns out they do support .NET Standard, but it's not obvious because they are crafting the \*.nuspec by hand and do not declare their dependencies. I filed Microsoft/perfview#716 against to help us consumers understand what they support.

**Comment by karolz-ms** [comment]
Just published ETW Input 1.4.0 that should support .NET Core 1.1 and 2.0
